code/modules/hivemind/mobs.dm

```diff
@@ -405,15 +411,15 @@
 	desc = "Once a man, now metal plates and tubes weave in and out of their oozing sores."
 	icon_state = "himan"
 	icon_dead = "himan-dead"
-	health = 120
-	maxHealth = 120
+	health = 330
```

Comment:
That's clearly too much, I got your point, but let's keep at 250. They have their ability that can be used effectively.

Reply:
They can still be damaged while faking their death. Would 280 seem better maybe? With the ability activating at under 160 hp?

Reply:
We should not increase their health too much by contributing guidelines.

Reply:
Will change the hp to 250 then

Reply:
Done so, along with fixing the attack message so it makes sense